ss/multiconn.go

```diff
@@ -0,0 +1,25 @@
+package ss
+
+import (
+	"strings"
+	"sync"
+)
+
+var routes struct {
+	sync.RWMutex
+	Target2Client map[string]string // map target ip to local tunnel port

```

Comment:
I would suggest we change `2` to `To` follow the same naming convention.

Reply:
Yes, modified.